runtime/secrets/secrets_test.go

```diff
@@ -34,9 +34,16 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 )
 
+const (
+	testNS              = "default"
+	githubAppPrivateKey = "-----BEGIN PRIVATE KEY-----\nMIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQC5"
```

Comment:
🤔

Reply:
I left this part as is because there are still some places in the factory's tests that use the GitHub App private key.

Reply:
I suppose this can be renamed since there's nothing really related to GitHub App here?

Reply:
Nevermind!